nominees/xr-accessibility-project.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "XR Accessibility Project",
+  "aliases": [
+    "XRAP"
+  ],
+  "description": "An index of resources for people creating accessible XR (extended reality) applications.",
+  "website": "https://github.com/XRAccessibility/xraccessibility.github.io",
+  "license": [
+    {
+      "spdx": "CC-BY-SA-4.0",
+      "licenseURL": "https://github.com/XRAccessibility/xraccessibility.github.io/blob/main/README.md"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 9,
+      "evidenceText": "Supports accessible communication, training, and education XR infrastructure so that people with disabilities can benefit from the new applications of these technologies."
+    },
+    {
+      "SDGNumber": 10,
+      "evidenceText": "Reduces inequality by ensuring equitable access to new technologies."
```

Comment:
I'm not sure access to new technologies is supported by the Targets for [SDG Goal 10](https://sdgs.un.org/goals/goal10). I think for this submission to have SDG relevance for Goal 10, it needs to be more specific about how exactly inequality is reduced. Framing it from the point of view of new technologies might not be helpful.

Reply:
Hi JW, I'm the submitter. How about this:

> The XR Accessibility Project seeks to reduce income inequality by enabling people with disabilities to take part in the hybrid workplace. XR offers new opportunities to work from home by enriching online communication and enabling remote piloting and operation of machinery. Ensuring that people with disabilities can participate in these opportunities should raise the employment rate for people with disabilities in the US from the meager 19.1 percent it stood at in 2021 (according to the [US Bureau of Labor Statistics](https://www.bls.gov/news.release/disabl.nr0.htm)). This is in line with Target 10.2 ("By 2030, empower and promote the social, economic and political inclusion of all, irrespective of age, sex, disability, race, ethnicity, origin, religion or economic or other status") and 10.3 ("Ensure equal opportunity and reduce inequalities of outcome, including by eliminating discriminatory laws, policies and practices and promoting appropriate legislation, policies and action in this regard.")

Reply:
Thanks @dylan-fox, looks good to me.